pkg/rancher-components/src/components/Form/LabeledInput/LabeledInput.vue

```diff
@@ -143,22 +146,22 @@ export default {
      * Emit on input with delay
      * Note: Arrow function is avoided due context binding
      */
-    delayInput(value) {
+    delayInput(value: string | number | Record<string, unknown>) {
       this.$emit('input', value);
     },
 
     onFocus() {
       this.onFocusLabeled();
     },
 
-    onBlur(event) {
+    onBlur(event: string | number | Record<string, unknown>) {
```

Comment:
As for `delayInput` this value can be only type of `string`.

Returned value is `void`